### PR TITLE
feat(config): add support for TechniSat On/Off switch flush mount, BJ 

### PR DESCRIPTION
**Pull request by Helidude**
Fix for issue #1597 missing support for TechniSat switch 0x0299-0x1a90-0x0002
Data from z-wavealliance.org

### CODE REVIEW

File: packages/config/config/devices/0x0299/1a90.json [hunk 1]
```diff
@@ -0,0 +1,133 @@
+// Technisat series switches
+// Single-Switch
+{
+	"manufacturer": "TechniSat",
+	"manufacturerId": "0x0299",
+	"label": "Single-Switch",
+	"description": "Single-Switch",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0299/1a90.json]:
```suggestion
	"label": "Single-Switch 0300/9499",
	"description": "On/Off switch flush mount, BJ",
```
Does that make sense to be a bit more precise?

File: packages/config/config/devices/0x0299/1a90.json [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "Central Scene notifications",
+			"description": "Setting this parameter to 0 will disable central scene notifications for multiple (2x-5x button presses)..",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0299/1a90.json]:
```suggestion
			"label": "Central Scene: Multiple Button Presses",
```
seems more concise(?)

File: packages/config/config/devices/0x0299/1a90.json [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+			"options": [
+				{
+					"label": "Disable Central Scene",
+					"value": 0
+				},
+				{
+					"label": "Enable Central Scene",
+					"value": 1
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0299/1a90.json]:
```suggestion
			"options": [
				{
					"label": "Disabled",
					"value": 0
				},
				{
					"label": "Enabled",
					"value": 1
				}
			]
```

File: packages/config/config/devices/0x0299/1a90.json [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+		},
+		"2": {
+			"label": "Report interval",
+			"description": "3 to 8640 - Interval of unsolicited reports in 10s increments, a value of 3 means 30 seconds (30 seconds - 1 day) 0 - Disable unsolicited meter reports of current wattage.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0299/1a90.json]:
```suggestion
			"description": "Interval of unsolicited meter reports in 10s increments; 0 to disable.",
```

File: packages/config/config/devices/0x0299/1a90.json [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+			"minValue": 0,
+			"maxValue": 8640,
+			"defaultValue": 3,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"3": {
+			"label": "Energy meter report interval",
+			"description": "Interval of active energy meter reports in minutes. - 10 to 30240 - Interval of active energy meter unsolicited reports in minutes (10 minutes - 3 weeks) - 0 - Disable unsolicited meter reports of active energy.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0299/1a90.json]:
```suggestion
			"description": "Interval of active energy meter reports in minutes; 0 to disable.",
```

File: packages/config/config/devices/0x0299/1a90.json [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "In this mode the top button will turn the output on, while the bottom button will turn the output off.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0299/1a90.json]:
```suggestion
					"label": "Top: turn off, Bottom: turn on",
```

File: packages/config/config/devices/0x0299/1a90.json [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+					"value": 0
+				},
+				{
+					"label": "In this mode any button press will toggle the output - when on it will be turned off, when off it will be turned on.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0299/1a90.json]:
```suggestion
					"label": "Toggle the output",
```

File: packages/config/config/devices/0x0299/1a90.json [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+					"value": 1
+				}
+			]
+		},
+		"5": {
+			"label": "Input S Mode",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0299/1a90.json]:
```suggestion
			"label": "Input S: Button Mode",
```

File: packages/config/config/devices/0x0299/1a90.json [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+			"options": [
+				{
+					"label": "Use this value if a toggle switch is connected to extension input S.",
+					"value": 0
+				},
+				{
+					"label": "Use this value if a push button switch is connected to extension input S.",
+					"value": 1
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0299/1a90.json]:
```suggestion
			"options": [
				{
					"label": "Toggle Switch",
					"value": 0
				},
				{
					"label": "Momentary Switch",
					"value": 1
				}
			]
```

File: packages/config/config/devices/0x0299/1a90.json [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+		},
+		"6": {
+			"label": "Input S Scenes",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0299/1a90.json]:
```suggestion
			"label": "Input S: Central Scene Events for multiple button presses",
```

File: packages/config/config/devices/0x0299/1a90.json [hunk 11]
```diff
@@ -0,0 +1,133 @@
...
+				},
+				{
+					"label": "Multiple button presses (2x-5x) on extension input S will trigger events of central scene 2.",
+					"value": 2
+				},
+				{
+					"label": "Multiple button presses (2x-5x) on extension input S will trigger events of central scene 3 - an additional central scene exclusive to extension input S.",
+					"value": 3
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0299/1a90.json]:
```suggestion
			"options": [
				{
					"label": "Central Scene 1",
					"value": 1
				},
				{
					"label": "Central Scene 2",
					"value": 2
				},
				{
					"label": "Central Scene 3 (exclusive to Input S)",
					"value": 3
				}
			]
```

File: packages/config/config/devices/0x0299/1a90.json [hunk 12]
```diff
@@ -0,0 +1,133 @@
...
+				},
+				{
+					"label": "Use this value if a push button switch is connected to extension input S.",
+					"value": 1
+				}
+			]
+		},
+		"6": {
+			"label": "Input S Scenes",
+			"description": "Scene mode trigger type",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0299/1a90.json]:
```suggestion
			"description": "Which Central Scene to trigger on multiple button presses (2x-5x) of extension input S",
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
The wording was a bit verbose sometimes. Labels should be pretty short in general.

File: packages/config/config/devices/0x0299/1a90.json (hunk 1 above)
Comment (Helidude) [reply packages/config/config/devices/0x0299/1a90.json]: That makes sense

File: packages/config/config/devices/0x0299/1a90.json (hunk 2 above)
Comment (Helidude) [reply packages/config/config/devices/0x0299/1a90.json]: I used the other configs as an template whith that exact wording. But happy to change it if this is the way!

**Comment by Helidude** [comment]
@AlCalzone For the rest of the changes.. I copy/pasted from z-wavealliance product descriptions. Is that not common practice? 
Please let me know, and I'm happy to change the rest to meet your practices

**Comment by blhoward2** [comment]
We change a lot of the zwa provided fields because they're written by engineers. They tend to be very verbose.

File: packages/config/config/devices/0x0299/1a90.json (hunk 2 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0299/1a90.json]: We're in the process of cleaning up old files, so that happens sometimes.

**Comment by Helidude** [comment]
All good. Committed suggested changes

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**